### ISSUE TITLE
Dependabot keeps opening issues about not finding a dependency

### ISSUE THREAD

**Issue by ElvenSpellmaker** [dependabot/feedback#666]
Hey,

Recently in one private repo of ours Dependabot has taken to opening issues that it can't reach our linting dependencies despite them being public repos:
![Screenshot_20190910-073919__01](https://user-images.githubusercontent.com/2286713/64590137-be268600-d39e-11e9-858c-28717ae4ffc0.jpg)
![Screenshot_20190910-073911__01](https://user-images.githubusercontent.com/2286713/64590164-ce3e6580-d39e-11e9-8f23-7cb71f0701f0.jpg)

Here's an excerpt of our `composer.json`:
```
		"ve-interactive/php-sniffer-rules": "dev-fix/sniffer-v3",
		"ve-interactive/php-mess-detector": "dev-fix/update-md"
... 
	"repositories": [
		{
			"type": "vcs",
			"url": "git@github.com:ve-global/PhpSnifferRules.git"
		},
		{
			"type": "vcs",
			"url": "git@github.com:ve-global/VeMessDetector.git"
		}
	],
```

I tried contacting the bot (as told to do so in the issue) but have no response yet, perhaps the bot can't work with private repos in that way?

I'm wondering if it's an auth thing to the API?

**Comment by greysteil** [comment]
It looks like those branches genuinely don't exist (e.g., see https://github.com/ve-global/PhpSnifferRules/branches) so I think Dependabot is correct to raise an error saying it can't fetch that branch, no?

Less good that the issue detail keeps flapping...

**Comment by ElvenSpellmaker** [comment]
@greysteil You're correct the branches don't seem to exist!

Weird how `composer install` still works, I reckon the build agents and my local have caches and so are using that instead of getting it from the remote.

Perhaps dependabot has caches and sometimes it can see the braches and sometimes it can't depending on how your infra for dependabot works.

EDIT: Yeah, I can see you're indeed correct that I'm getting it from cache:
```
  - Installing ve-interactive/php-mess-detector (dev-fix/update-md 35f2520): Cloning 35f2520244 from cache
...
  - Installing ve-interactive/php-sniffer-rules (dev-fix/sniffer-v3 f97aa9e): Cloning f97aa9ec02 from cache
```

Sorry, but the flapping does seem odd, as I said I reckon it'd to do with caches on some of the 'agents' that dependabot probably uses, and so sometimes it can see the dependencies and othertimes it can't!

I'll switch the `composer.json` to use the correct tags, thanks! 🤦‍♂

**Comment by greysteil** [comment]
Awesome! I'll have a look into the flapping.
